### PR TITLE
[candi] redos 8.0 support

### PR DESCRIPTION
**Pull request by borg-z**
## Description
Added support for Red OS 8.0

## Why do we need it, and what problem does it solve?
This update ensures compatibility with the latest version of Red OS

## Why do we need it in the patch release (if we do)?
Not necessarily.

## What is the expected result?
The system should now correctly detect and install necessary packages for Red OS 8.0. 

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: candi
type: feature
summary: Added support for Red OS 8.0.
impact: Support for Red OS 8.0 ensures compatibility with the latest OS version, providing updated packages and configurations.
impact_level: default
```

### CODE REVIEW

**Review by RomanenkoDenys — APPROVED** [review]

**Comment by RomanenkoDenys** [comment]
Please run e2e

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by borg-z** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.25 9298746312 9298746312-1-con-1-25 /sys/deckhouse-oss/install:pr8530 redos@178.154.226.134

**Comment by borg-z** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.25 9298746312 9298746312-1-con-1-25 /sys/deckhouse-oss/install:pr8530 redos@178.154.226.134

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Review by sprait — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @borg-z. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/_data/supported_versions.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ osDistributions:
   ubuntu:
     name: Ubuntu
     url: https://ubuntu.com/download/server
-  '_redos':
+  redos:
```
Comment (z9r5) [inline docs/documentation/_data/supported_versions.yml:24]:
```suggestion
  '_redos'::
```
It was a dirty trick to proper items sorting  :)

File: docs/documentation/_data/version_map_addition.yml [hunk 2]
```diff
@@ -6,8 +6,9 @@
   rocky:
     '8':
     '9':
-  '_redos':
+  redos:
```
Comment (z9r5) [inline docs/documentation/_data/version_map_addition.yml:9]:
```suggestion
  '_redos':
```

File: docs/documentation/_data/supported_versions.yml (hunk 1 above)
Comment (z9r5) [reply docs/documentation/_data/supported_versions.yml:24]: @borg-z JFI

File: docs/documentation/_data/supported_versions.yml [hunk 3]
```diff
@@ -21,7 +21,7 @@ osDistributions:
   ubuntu:
     name: Ubuntu
     url: https://ubuntu.com/download/server
-  '_redos':
+  '_redos'::
```
Comment (z9r5) [inline docs/documentation/_data/supported_versions.yml:24]:
```suggestion
  '_redos':
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/_data/supported_versions.yml (hunk 1 above)
Comment (borg-z) [reply docs/documentation/_data/supported_versions.yml]: I see, I thought it was some kind of typo.

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by z9r5** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.25 9345144141 9345144141-1-con-1-25 /sys/deckhouse-oss/install:pr8530 redos@158.160.105.216

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Pull request merged**